.github/workflows/builds.yaml

```diff
@@ -380,3 +380,51 @@ jobs:
             builddir/meson-logs/
             /var/log/messages
             /var/log/syslog
+
+  subprojects:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout HSE
+        uses: actions/checkout@v2
+
+      - name: Checkout hse-python
+        uses: actions/checkout@v2
+        with:
+          repository: hse-project/hse-python
+          path: subprojects/hse-python
+
+      - name: Post-checkout
+        run: |
+          for b in hse-python; do
+            git -C "subprojects/$b" checkout ${{ github.head_ref }} || true
+          done
+
+      - name: Initialize
+        run: |
+          sudo apt-get update
+          sudo apt-get install build-essential ninja-build pkg-config
+          sudo python3 -m pip install meson Cython
+
+      - name: Setup
+        run: |
+          meson builddir -Dwrap_mode=forcefallback -Dtools=enabled \
+            -Ddocs=disabled -Dbindings=all -Dycsb=true -Dwerror=true
+
+      - name: Build
+        run: |
+          ninja -C builddir
+
+      - name: Test
+        run: |
+          ulimit -c unlimited
+          meson test -C builddir --setup=ci --print-errorlogs --no-stdsplit
+
+      - uses: actions/upload-artifact@v2
+        if: failure()
+        with:
+          name: ${{ github.job }}
+          path: |
+            builddir/meson-logs/
+            /var/log/messages
+            /var/log/syslog
```

Comment:
Would probably be good to collect only the logs generated by the test.  Maybe you can run a command to rotate the log file, or grab the before and after logs and trim the duplicate entries?

Reply:
I see what you are saying. You want to truncate /var/log/messages and /var/log/syslog prior to running tests?

Reply:
Well, ideally we wouldn't truncate it, but if this is running on a CI system that just came into existance for this test (or just this session?) then maybe it doesn't matter...

Reply:
It is spun up specifically for this purpose and then deallocated.